README.md

```diff
@@ -1 +1,46 @@
 # Open city sign-ups
+
+## Development
+
+### Install pip-tools
+
+* Run `pip install pip-tools`
+
+### Creating Python requirements files
+
+* Run `pip-compile requirements.in`
+* Run `pip-compile requirements-dev.in`
+
+### Updating Python requirements files
+
+* Run `pip-compile --upgrade requirements.in`
+* Run `pip-compile --upgrade requirements-dev.in`
+
+### Installing Python requirements
+
+* Run `pip-sync requirements.txt requirements-dev.in`
+
+### Database
+
+To setup a database compatible with default database settings:
+
+Create user and database
+
+    sudo -u postgres createuser -P -R -S open_city_signups  # use password `open_city_signups`
+    sudo -u postgres createdb -O open_city_signups open_city_signups
+
+Allow user to create test database
+
+    sudo -u postgres psql -c "ALTER USER open_city_signups CREATEDB;"
+
+### Daily running
+
+* Set the `DEBUG` environment variable to `1`.
+* Run `python manage.py migrate`
+* Run `python manage.py runserver 0:8000`
+
+## Running tests
+
+* Set the `DEBUG` environment variable to `1`.
+* Run `pytest`.
+>>>>>>> 7e41476... Project base
```

Comment:
A bit of git stuff there

Reply:
Ah, deleted in the next commit, my bad :slightly_smiling_face:

Reply:
Nah, it shouldn't have been like that, fixed it properly now :)